WBooks/Books Table/BooksViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  BooksViewModel.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 8/24/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+import ReactiveSwift
+import Result
+
+class BooksViewModel {
+    
+    // MARK: - Properties
+    
+    private let mutableBooks = MutableProperty([Book]())
+    private let _error: Signal<NSError, NoError>.Observer
+    let books: Property<[Book]>
+    let errorSignal: Signal<NSError, NoError>
```

Comment:
I prefer to call it `errors` (like you do with `books`) since by the type we already know it's a signal